test/demo/collections/main.js

```diff
@@ -131,8 +131,8 @@
                     var box = new OpenSeadragon.Rect(margins.left, margins.top,
                         $('#contentDiv').width() - (margins.left + margins.right),
                         $('#contentDiv').height() - (margins.top + margins.bottom));
-
-                    self.viewer.drawer.debugRect(box);
+                    // if drawDebuggingRect is implemented, use it to show the box
```

Comment:
Yes, the comment is there because the `DrawerBase` API throws a warning if this isn't implemented; the `HTMLDrawer` doesn't implement it. I feel like it isn't critical which is why I made it a warning only. The comment is there just to indicate that it may not be implemented depending on which drawer is used. I'm happy to clarify however makes the most sense. I'm modifying the comment slightly for clarity but it might need more work.